src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -148,7 +148,7 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
                 } else {
                     DefaultComboBoxModel(arrayOf(defaultMessage))
                 }
-                row("Project") {
+                row("Project:") {
```

Comment:
Victor had feedback in the blitz that Project is singular but Environments is plural. We should stick with one, maybe singular? @launchdarkly/docs-reviewers any thoughts

Reply:
yea singular, and the Environments label is updated in https://github.com/launchdarkly/ld-intellij/pull/26